helpers/scripts/composer_dag_send_email.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# - Wait until the DAG is initialized
+n=0
+until [[ $n -ge $4 ]]
+do
+  status=0
+  gcloud beta composer environments run $1 --location $2 dags list 2>&1 | grep $3 && break
+  status=$?
+  n=$(($n+1))
+  sleep $5
+done
+
+# - Unpause the DAG if paused
+gcloud beta composer environments run $1 --location $2 dags unpause -- $3
+
+# - Run the DAG "semd_email". The email contains commit ID.
+gcloud beta composer environments run $1 --location $2 dags trigger -- $3 --conf '{"key":"'${6}'"}'
```

Comment:
should we change "key" to "version"? (here and in the dag).

Reply:
It only works with "key" and its value for some reaason. Initially tried "Commit ID" but it did not work.

Reply:
loool. so weird

Reply:
I am double checking it right now

Reply:
It actually worked when I double-checked it. I've given a proper name to the parameter. Yeah, weird